test/DynamoCoreTests/CustomNodes.cs

```diff
@@ -1375,5 +1375,20 @@ public void InputNameWithSpacesShouldCauseWarningInGraph()
             // Despite the invalid input symbol, the custom node should still execute and return the correct value
             AssertPreviewValue("f91658aeafe84cbaa39d370dec283b53", 8.0);
         }
+
+        [Test]
+        public void LooseCustomNodeShouldNotHavePackageInfoOrPackageMember()
+        {
+            //load any loose custom node
+            var dynFilePath = Path.Combine(TestDirectory, @"core\CustomNodes\invalidInputName.dyn");
```

Comment:
Shouldn't this be a dyf?

Reply:
yes, thanks for catching, I am still working on the tests.